### DISCUSSION TITLE
30% Gas Fees for dApp Owners

### DISCUSSION THREAD

**Discussion by idea404** [near/near-sdk-js#47] [General]
Are the 30% gas fees earnt by the dApp sent to the account that sent the wasm to the JSVM account? Or are they stored in the JSVM account and the JSVM smart contract has a data container that keeps track of how much NEAR what accounts own?

**Comment by ailisp** [comment]
Since we are moving to standalone, we now have protocol-provided 30% gas fee to dApp owners!
